### PR TITLE
Importing Victor's vocabulary document as ASCIIDoc

### PR DESCRIPTION
**Pull request by castelao**
@vturpin and @justinbuck , I'll let for you to verify if the conversion didn't miss anything. I don't have the time now to deal with style adjustments. I'm requesting a review for the content only.

### CODE REVIEW

File: src/OG_Vocabulary.adoc [hunk 1]
```diff
@@ -0,0 +1,1411 @@
...
+
+|DRUCK | |SDN:L22::TOOL0837 | |approved |Already used by OG
+
+|DRUCK_10153PSIA | | | | |
+
+|DRUCK_2900PSIA | | | | |Already used by OG
+
+|DURA | | | | |
+
+|DISSOLVED_OXYGEN_SENSOR | | | | |Already used by OG
```
[the remark this answers is not in the document]
Comment (castelao) [reply src/OG_Vocabulary.adoc:1194]: I agree, but I suggest that to just reproduce the document here, and once it is merged we start the discussions and improvements.

File: src/OG_Vocabulary.adoc [hunk 2]
```diff
@@ -0,0 +1,1411 @@
...
+| |WACoastCascadia |Washington Coast Cascadia |LINESTRING ( -128.00
+47.00, -124.00 47.00) |
+
+| |WACoastNANOOS |Washington Coast NANOOS |LINESTRING ( -125.17 47.87,
+-127.00 47.00) |
+
+| |WSC |Western Svalbard Current |LINESTRING (0 77.25,12 77.25) |
+|===
+
+_Master list_: None·
```
[the remark this answers is not in the document]
Comment (castelao) [reply src/OG_Vocabulary.adoc:564]: Removed. thanks

File: src/OG_Vocabulary.adoc [hunk 3]
```diff
@@ -0,0 +1,1411 @@
...
+_Note_ : Phase calculation methods need publishing as a best practice
+document separately to the OG1.0 ToR.
+
+[#anchor-19]####Data_mode
+
+[cols=",,",]
+|===
+|_Collection_ |_definition_ |_Status_
+
+|R a|
```
[the remark this answers is not in the document]
Comment (castelao) [reply src/OG_Vocabulary.adoc:1069]: removed

File: src/OG_Vocabulary.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (castelao) [reply src/OG_Vocabulary.adoc:1193]: @soerenthomsen , please let me know if you would be fine with that.

[the remark this answers is not in the document]
Comment (soerenthomsen) [reply src/OG_Vocabulary.adoc:1193]: Agreed. I created an issue #40

**Comment by justinbuck** [comment]
@soerenthomsen  yes http://vocab.nerc.ac.uk/collection/L05/current/351/ is the genraic dosovled gas sensor term we have, but would be better to map to an L22 term http://vocab.nerc.ac.uk/search_nvs/L22/?searchstr=oxygen&options=identifier,preflabel,altlabel,status_accepted&rbaddfilter=inc&searchstr2=

**Comment by justinbuck** [comment]
I have raised this with SDN (Mark Hebden), we will review and feedback.

**Comment by castelao** [comment]
@justinbuck , good point. It's a draft state now. Once you or @vturpin approve the review I'll merge it into the main branch and you will be able to start new branches editing it. Thanks,

**Review by justinbuck — APPROVED** [review]
Happy to be committed in this draft state (the title makes clear that it is a draft document) ahead of an OG meeting to progress the content.

**Pull request merged**